test/db/cmd/cmd_i

```diff
@@ -4643,7 +4643,7 @@ e bin.str.purge=all,!0x413220-0x413235
 EOF
 EXPECT=<<EOF
 --1--
-1049 0x00012420 0x00412420 5   6    .text              ascii   AWAVA
+1100 0x00012420 0x00412420 5   6    .text              ascii   AWAVA
```

Comment:
And here - a difference.

Reply:
[ls2.diff.txt](https://github.com/rizinorg/rizin/files/7528313/ls2.diff.txt)


```
+145  0x00002ae5 0x00402ae5 7   8    .text              ibm037  6çiGYkW
+151  0x00003381 0x00403381 6   7    .text              ibm037  <iXY7¶
+152  0x000033b9 0x004033b9 10  11   .text              ibm037  6çiA<iXY]ù
+171  0x00003def 0x00403def 6   7    .text              ibm037  <iXYr;
+172  0x00003e72 0x00403e72 4   5    .text              ibm037   YYX
+176  0x00003edf 0x00403edf 6   7    .text              ibm037  <iXYr¿
+177  0x00003f84 0x00403f84 6   7    .text              ibm037  6iÕYMV
+184  0x00004484 0x00404484 4   5    .text              ibm037  {YWS
+185  0x0000452f 0x0040452f 6   7    .text              ibm037  <iXYrú
+186  0x00004555 0x00404555 6   7    .text              ibm037  <iXYty
+223  0x00005463 0x00405463 5   6    .text              ibm037  à0YFJ
+253  0x0000611f 0x0040611f 6   7    .text              ibm037  <iXYZB
+255  0x0000616b 0x0040616b 15  16   .text              ibm037  Í÷çiú<iX$) *ZMC
+266  0x0000676f 0x0040676f 6   7    .text              ibm037  KcS4cB
+283  0x00006bbf 0x00406bbf 10  11   .text              ibm037  {<iX<iVYE¯
+298  0x00006f27 0x00406f27 6   7    .text              ibm037  çiBYA³
+299  0x00006f52 0x00406f52 7   8    .text              ibm037  V<iXYE§
+327  0x00007544 0x00407544 7   8    .text              ibm037  <iW(iXY
+358  0x00007c32 0x00407c32 13  14   .text              ibm037  ­aSØØØØÈYiJAZ
+376  0x00008393 0x00408393 6   7    .text              ibm037  çiGYV¤
+382  0x000087bf 0x004087bf 13  14   .text              ibm037  ­aSØØØØÈYiJAZ
+385  0x0000893a 0x0040893a 9   10   .text              ibm037  <iSçiFY$ó
+386  0x0000898d 0x0040898d 6   7    .text              ibm037  ñiEZWÛ
+425  0x00009030 0x00409030 6   7    .text              ibm037  çiCYQl
+442  0x000093d0 0x004093d0 7   8    .text              ibm037  çiC<iXY
+445  0x00009440 0x00409440 9   10   .text              ibm037  çiC<iXYE±
+485  0x00009fa5 0x00409fa5 12  13   .text              ibm037  çiB<iW<i7Y\rÚ
+490  0x0000a0a5 0x0040a0a5 6   7    .text              ibm037  çiEYca
+496  0x0000a471 0x0040a471 4   5    .text              ibm037  GYIa
+497  0x0000a4b0 0x0040a4b0 6   7    .text              ibm037  <iWYqb
+600  0x0000be98 0x0040be98 4   5    .text              ibm037  RIZº
+631  0x0000c7a9 0x0040c7a9 8   9    .text              ibm037  KiOcB\bñ»
+640  0x0000ca5e 0x0040ca5e 9   10   .text              ibm037  <iWçiCY7ï
+647  0x0000cbe7 0x0040cbe7 6   7    .text              ibm037  <iXYjï
+651  0x0000cc73 0x0040cc73 6   7    .text              ibm037  <iWYNì
+653  0x0000ccc9 0x0040ccc9 13  14   .text              ibm037  {) *CÃ°<iXY8ì
+686  0x0000d509 0x0040d509 18  19   .text              ibm037  çiZçi0<i2çJYñiGçià
+814  0x0000ea23 0x0040ea23 5   6    .text              ibm037  <iXYE
+821  0x0000eb68 0x0040eb68 10  11   .text              ibm037  çcD $) *CY
+826  0x0000ec1c 0x0040ec1c 6   7    .text              ibm037  i}LYc0
+836  0x0000ed43 0x0040ed43 7   8    .text              ibm037  çcDçCYL
+861  0x0000f1b2 0x0040f1b2 6   7    .text              ibm037  <iYZyÚ
+930  0x0000f984 0x0040f984 9   10   .text              ibm037  ´ihhh7Óâý
+955  0x0000ffa4 0x0040ffa4 8   9    .text              ibm037  i87²i8A8
+957  0x0001018d 0x0041018d 4   5    .text              ibm037  IZNÙ
+958  0x00010195 0x00410195 4   5    .text              ibm037  IZIÚ
+959  0x0001024e 0x0041024e 4   5    .text              ibm037  iFFd
+1054 0x0001109f 0x0041109f 6   7    .text              ibm037  çiFZRÚ
+1061 0x00011464 0x00411464 4   5    .text              ibm037  KZTÚ
+1070 0x00011a94 0x00411a94 4   5    .text              ibm037  KZTÚ
+1071 0x00011d65 0x00411d65 13  14   .text              ibm037  çiO3wÍTçiPY*\n
+1082 0x00012072 0x00412072 4   5    .text              ibm037  iGYg
+1094 0x00012357 0x00412357 4   5    .text              ibm037  iGYK
```